portfolio/src/main/webapp/script.js

```diff
@@ -12,20 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random fact to the page.
- */
-function addRandomFact() {
-  const facts =
-      ["I used to be a springboard diver until I hit my face on the diving board :(", 
-      "Zhajiangmian is my favorite food!", 
-      "I'm from Kansas!", 
-      "I'm a huge Marvel fan!"];
+function openCourses(evt, courseName) {
+    var i, tabcontent, tablinks;
```

Comment:
instead, use for (let i = 0, ...) directly.

Reply:
Done